lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -105,6 +106,30 @@ function runLighthouse(url, configPath, saveAssetsPath) {
 }
 
 /**
+ * Checks if the actual value matches the expectation. Does not recursively search. This supports
+ *    - Greater than/less than operators, e.g. "<100", ">90"
+ *    - Regular expressions
+ *    - Strict equality
+ *
+ * @param {*} actual
```

Comment:
nah it can be any, the Object.is catch-all does boolean and number expectations too, objects will also be passed in but will always be false